code/game/machinery/firealarm.dm

```diff
@@ -47,8 +57,16 @@
 	update_appearance(UPDATE_ICON)
 	myarea = get_area(src)
 	LAZYADD(myarea.firealarms, src)
+	radio = new(src)
+	radio.keyslot = new /obj/item/encryptionkey/headset_eng
+	radio.subspace_transmission = TRUE
+	radio.canhear_range = 0
+	radio.recalculateChannels()
+	STOP_PROCESSING(SSmachines, src) // I will do this
```

Comment:
This is unfortunate but there's nothing we can do as we don't have https://github.com/tgstation/tgstation/pull/49307